leg/assets/myolegs_assets.xml

```diff
@@ -21,14 +21,14 @@
             <geom margin="0.001" material="mat_myolegs" rgba="0.8 0.85 0.8 1" conaffinity="0" contype="0"/>
             <site size="0.001 0.005 0.005" group="3"/>
             <tendon rgba="0.95 0.3 0.3 1" width="0.001"/>
-            <default class="muscle">
+            <default class="myoleg_muscle">
                 <general biasprm="0.75 1.05 -1 400 0.5 1.6 1.5 1.3 1.2 0" biastype="muscle" ctrllimited="true" ctrlrange="0 1" dynprm="0.01 0.04 0 0 0 0 0 0 0 0" dyntype="muscle" gainprm="0.75 1.05 -1 400 0.5 1.6 1.5 1.3 1.2 0" gaintype="muscle"/>
             </default>
-            <default class="wrap">
+            <default class="myoleg_wrap">
                 <geom rgba=".5 .5 .9 0.5" group="3" contype="0" conaffinity="0"  type="cylinder"/>
             </default>
             <default class="coll">
-                <geom type="capsule" group="4" contype="1" conaffinity="0" condim="3" rgba="0.8 0.7 .5 .1" margin="0.001" material="MatSkin"/>
+                <geom type="capsule" group="4" contype="1" conaffinity="0" condim="3" rgba="0.8 0.7 .5 .1" margin="0.001" material="MatSkin_myoleg"/>
```

Comment:
sometime `myoleg` is used as prefix, sometime as suffix. Be consistent throughout the repo.

Reply:
https://github.com/MyoHub/myo_sim/issues/64

Reply:
tracked here https://github.com/MyoHub/myo_sim/issues/64